### PR TITLE
Use Fluo and Accumulo version installed for shaded jar

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: bin/run-test.sh [hunk 1]
```diff
@@ -38,8 +38,7 @@ if [ ! -d $FLUO_APP_LIB ]; then
   exit 1
 fi
 cp $STRESSO_JAR $FLUO_APP_LIB
-fluo_recipes_version=1.0.0-incubating-SNAPSHOT
-mvn dependency:get -Dartifact=org.apache.fluo:fluo-recipes-core:$fluo_recipes_version:jar -Ddest=$FLUO_APP_LIB
+mvn dependency:copy-dependencies  -DincludeArtifactIds=fluo-recipes-core -DoutputDirectory=$FLUO_APP_LIB
```
Comment (mikewalch) [inline bin/run-test.sh:41]: interesting.. does it copy the version set in your pom.xm?

File: conf/env.sh.example [hunk 2]
```diff
@@ -31,3 +31,8 @@ ITERATIONS=3
 SLEEP=30
 # Compact levels with less than the following possible nodes after loads
 COMPACT_CUTOFF=$((256**3 + 1))
+# To run map reduce jobs a shaded jar is built, the following properties
+# determine what versions of Fluo and Accumulo client libs end up in the shaded
+# jar.
+FLUO_VERSION=$($FLUO_HOME/bin/fluo version)
+ACCUMULO_VERSION=$(accumulo version)
```
Comment (mikewalch) [inline conf/env.sh.example:38]: Should mention in README.md that accumulo command should be on PATH. It already mentions that yarn and hadoop should be there.

File: conf/env.sh.example [hunk 3]
```diff
@@ -31,3 +31,8 @@ ITERATIONS=3
 SLEEP=30
 # Compact levels with less than the following possible nodes after loads
 COMPACT_CUTOFF=$((256**3 + 1))
+# To run map reduce jobs a shaded jar is built, the following properties
```
Comment (mikewalch) [inline conf/env.sh.example]: Could be `To run MapReduce jobs, a shaded jar is built. The following`

**Review by mikewalch — CHANGES_REQUESTED** [review]

**Comment by mikewalch** [comment]
I used `run-test.sh` to run stresso locally using Uno.  Everything worked for me.

File: bin/run-test.sh (hunk 1 above)
Comment (keith-turner) [reply bin/run-test.sh:41]: Yeah it uses the version in the pom.

**Review by mikewalch — APPROVED** [review]
+1

**Pull request merged**